recipe/build.sh

```diff
@@ -1,5 +1,15 @@
 #!/bin/bash
 
+set -ex
+
+if [ `uname` == Darwin ]; then
+    # patch for Mac OSX error:
+    # arg_int.c:89:8: error: call to undeclared library function 'toupper' with type 'int (int)'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
+    # if (toupper(*ptr++)!=toupper(X))
+    # arg_int.c:89:8: note: include the header <ctype.h> or explicitly provide a declaration for 'toupper'
+    sed -i'.bak' -e "33i #include <ctype.h>" src/arg_int.c
```

Comment:
to be honest, i didn't see on my phone while reviewing that this was effectively a patch.

This isn't typical for us since we prefer to be explicit about the patches with hopes tat they eventually get upstreamed. 

Would you consider a patch for this change?

Reply:
Sure! 
Let me rework it!

Reply:
The project has no updates since 2011 - https://sourceforge.net/projects/argtable/files/
I'm afraid it won't be updated with my improvement :-/